### PR TITLE
[KOA-6217] Bump to eslint-config-skyscanner v15, which comes with eslint 8

### PR DESCRIPTION
**Pull request by dominicfraser**
Brings in https://github.com/Skyscanner/eslint-config-skyscanner/releases/tag/v15.0.0.

This will be a major release due to: https://github.com/Skyscanner/stylelint-config-skyscanner/blob/main/docs/eslint-as-dependency.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 
 [Unreleased changes](./UNRELEASED.md)
 
+# 8.0.0
```
Comment (dominicfraser) [inline CHANGELOG.md:9]: Missed moving this over from UNRELEASED on the last release, was focusing on the Release Notes in GitHub 🤦

File: UNRELEASED.md [hunk 2]
```diff
@@ -2,9 +2,9 @@
 
 _If `@skyscanner/eslint-config-skyscanner` has a major version change then `@skyscanner/stylelint-config-skyscanner` should also have a major version change, as explained in [the docs](./docs/eslint-as-dependency.md)_
 
-# 8.0.0
+# 9.0.0
```
Comment (dominicfraser) [inline UNRELEASED.md:5]: New UNRELEASED, almost the same as before

File: docs/release.md [hunk 3]
```diff
@@ -17,5 +17,5 @@ Want to test your changes before making it stable and available to the general t
 
 To do this on your BRANCH or if already merged to main:
 - Run `npm version <provide a custom version with the alpha or beta name e.g. 10.0.0-beta.1>`. This will create a new version in the `package.json` file and create a tagged commit (you don't need to commit this).
-- Run `npm publish --tag alpha|beta` - use the correct tag depending on if you want to publish and alpha or a beta.
+- Run `npm publish --registry=https://registry.npmjs.org/ --tag alpha|beta` - use the correct tag depending on if you want to publish and alpha or a beta.
```
Comment (dominicfraser) [inline docs/release.md:20]: Helps with copying and pasting these commands

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@skyscanner/stylelint-config-skyscanner",
-  "version": "7.1.0",
+  "version": "8.0.0",
```
Comment (dominicfraser) [inline package.json:3]: Something has gone weird with the last release, as this should already be in main

File: package.json [hunk 5]
```diff
@@ -40,7 +40,7 @@
     "*.js": "npm run lint:fix"
   },
   "dependencies": {
-    "@skyscanner/eslint-config-skyscanner": "^14.0.0",
+    "@skyscanner/eslint-config-skyscanner": "^15.0.0",
```
Comment (dominicfraser) [inline package.json:43]: This is the actual change

**Review by olliecurtis — APPROVED** [review]

**Pull request merged**